WorkspaceView.py

```diff
@@ -942,6 +946,44 @@ def newWorkspaceBtnClicked(self):
                 workspaceName, workspaceDesc, workspaceType, workspaceUrl
             )
 
+    def get_server_package_file(self):
+        remote_version_url = "https://raw.githubusercontent.com/Ondsel-Development/Ondsel-Lens/master/package.xml"
+        response = requests.get(remote_version_url)
+        if response.status_code == 200:
+            return response.text
+        return None
+
+    def get_local_package_file(self):
+        local_version_url = f"{modPath}/package.xml"
+        try:
+            with open(local_version_url, "r") as file_:
+                return file_.read()
+        except FileNotFoundError:
+            pass
+        return None
+
+    def get_version_from_package_file(self, packageFileStr):
+        if packageFileStr is None:
+            return None
+
+        lines = packageFileStr.split('\n')
+        for line in lines:
+            if "<version>" in line:
+                version = line.strip().lstrip("<version>").rstrip("</version>")
+                return version
+
+    def check_for_update(self):
+        local_version = self.get_version_from_package_file(self.get_local_package_file())
+        remote_version = self.get_version_from_package_file(self.get_server_package_file())
+
+        if local_version and remote_version and local_version != remote_version:
+            self.form.updateAvailable.setText(f"<a href=\"https://github.com/Ondsel-Development/Ondsel-Lens/blob/master/changeLog.md">Ondsel Lens v{remote_version} available!</a>")
```

Comment:
Let's keep all the URL strings together at the top of the file.  It makes the code a bit tidier and easier to maintain if locations change.

FYI, the URLs at the top of are 'http'  and I think should be https.